tests/snappi_tests/bgp/files/bgp_convergence_helper.py

```diff
@@ -828,7 +827,7 @@ def create_v4_topo():
                 route_range = bgpv4_peer.v4_routes.add(
                     name="Network_Group%d" % i)
                 route_range.addresses.add(
-                    address='200.1.0.1', prefix=32, count=number_of_routes)
+                    address='200.1.0.1', prefix=32, count=routes)
```

Comment:
why is this a triple nested function instead of 3 separate functions?

Reply:
no specific reason, it was written intitially like that for easier understanding.